pkg/libvirttools/virtualization.go

```diff
@@ -139,8 +140,9 @@ func (v *VirtualizationTool) processVolumes(mounts []*kubeapi.Mount, domXML stri
 	}
 
 	for _, mount := range mounts {
+		volumeName := containerName + "_" + strings.Replace(*mount.ContainerPath, "/", "_", -1)
```

Comment:
In this situation we should then verify if it's equal to `""`.
Is there any possibility if ContainerPath could be `nil`?
Is this really needed for POC?